hvpy/facade.py

```diff
@@ -27,3 +27,106 @@ def getJP2Image(
     """
     params = getJP2ImageInputParameters(date=date, sourceId=sourceId, jpip=jpip, json=json)
     return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getJP2HeaderInputParameters)
+def getJP2Header(
+    id: int,
+    callback: Optional[str] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Get the XML header embedded in a JPEG2000 image. Includes the FITS header
+    as well as a section of Helioviewer-specific metadata.
+
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getJP2Header
+    >>> getJP2Header(id=9838343,callback="xml_header")
+    'xml_header('<?xml version="1.0" encoding="utf-8"?><meta><fits><SIMPLE>1</SIMPLE><BITPIX>16</BITPIX>...'
+    """
+    params = getJP2HeaderInputParameters(id=id, callback=callback)
+    return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getJPXClosestToMidPointInputParameters)
+def getJPXClosestToMidPoint(
+    startTimes: List[datetime],
+    endTimes: List[datetime],
+    sourceId: int,
+    linked: bool = True,
+    verbose: bool = False,
+    jpip: bool = False,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Generate and (optionally) download a custom JPX movie of the specified
+    datasource with one frame per pair of startTimes/endTimes parameters.
+
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getJPXClosestToMidPoint
+    >>> from datetime import datetime
+    >>> getJPXClosestToMidPoint(startTimes=[datetime(2014, 1, 1, 0, 0, 0), datetime(2014, 1, 1, 2, 3, 3)], endTimes=[datetime(2014, 1, 1, 0, 45, 0), datetime(2014, 1, 1, 2, 33, 3)], sourceId=14, linked=False, verbose=False, jpip=True)
+    'jpip://helioviewer.org:8090/movies/SDO_AIA_335_F2013-12-31T18.30.00Z_T2013-12-31T21.03.03ZCMP.jpxmid'
```

Comment:
```suggestion
    'jpip://helioviewer.org:8090/movies/SDO_AIA_335_F2014-01-01T00.00.00Z_T2014-01-01T02.33.03ZCMP.jpxmid'
```
The CI says this is the value

Reply:
We can do this, but I copy-pasted the value from the python terminal, I am still confused.